src/app/(privateProfile)/calculator/page.tsx

```diff
@@ -1,9 +1,433 @@
-function Calculator() {
+"use client";
+
+import React, { useState, useEffect } from "react";
+
+import { AiFillInfoCircle } from "react-icons/ai";
+import { CgClose } from "react-icons/cg";
+
+import style from "./calculator.module.scss";
+
+export default function Calculator() {
+	const [value, setValue] = useState<number>(0);
+	const [isVisibleInfo, setIsVisibleInfo] = useState(false);
+	const [isMobile, setIsMobile] = useState(window.innerWidth <= 425);
+
+	const formatNumber = (num: number) => {
+		return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, " ");
+	};
+	const handleVisibleInfo = () => {
+		setIsVisibleInfo(true);
+	};
+	const handleCloseInfo = () => {
+		setIsVisibleInfo(false);
+	};
+	const handleSubmit = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
event: MouseEvent<HTMLButtonElement>

better using react types without using React object declarating. can you correct it here and in other places?